liquibase-cli/src/test/groovy/liquibase/integration/commandline/LiquibaseCommandLineTest.groovy

```diff
@@ -3,10 +3,12 @@ package liquibase.integration.commandline
 
 import liquibase.command.CommandBuilder
 import liquibase.configuration.ConfigurationDefinition
-import picocli.CommandLine
+import liquibase.logging.LogMessageFilter
```

Comment:
@nvoxland, can you please confirm that we want to swap out piccoli.CommandLine in favor of using liquibase.logging.LogMessageFilter? I ask because there are plans to deprecate the LogMessageFilter class.

CC @nikosmoum

Reply:
That is the right class to be importing, and also part of why this PR won't be needed if/when #2777 goes through. I just hadn't closed this one yet since whether we want to do this or not will depend on what happens with 2777